### DISCUSSION TITLE
`Enabled/Disabled` command in VS Code does nothing

### DISCUSSION THREAD

**Discussion by brettcannon** [community/community#38218] [Copilot Conversations]
Under VS Code 1.74-insiders, the `Configure Enabled/Disabled` doesn't seem to do anything.

**Comment by iamcjwilliams** [comment]
@brettcannon Are you experiencing an error with this issue?

**Reply by brettcannon** [reply]
No error, the command just doesn't seem to do anything. I would have expected _some_ action (whether it's opening my `settings.json`, some UI to help me disable Copilot for Markdown file, etc.). Otherwise I didn't have a way I could find via the UI to shut off Markdown suggestions until I dug into the contribution list in the extension list.
